core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/puppeteer-utils.js

```diff
@@ -101,7 +101,7 @@ module.exports = class baseUser {
   async clickOn(selector) {
     try {
       const [button] = await this.page.$x(
-        `\/\/*[contains(text(), '${selector}')]`);
+        `\/\/*[contains(text(), normalize-space('${selector}'))]`);
```

Comment:
Where is normalize-space defined? (If it's not in the codebase then there should be a comment here to explain.)

Reply:
@seanlip `normailze-space` is a x-path function which strips the trailing spaces. added a comment for explaining the fucntion

Reply:
The name does convey that, but you should point to a link that documents it.

Reply:
@seanlip Please check once now! added a link for the normalize function documentation.